cdap-docs/admin-manual/source/security/authorization.rst

```diff
@@ -246,14 +245,14 @@ Dataset Modules
    * - Operation
      - Privileges Required
    * - Deploy
-     - *WRITE* (on the namespace)
+     - *ADMIN*
    * - Delete
      - *ADMIN*
    * - Delete-all in the namespace
-     - *ADMIN* (on the namespace)
-   * - List
-     - Only returns those artifacts on which user has at least one of *READ, WRITE, EXECUTE,* or *ADMIN*
-   * - View
+     - *ADMIN* on all dataset modules in the namespace
+   * - List/View
+     - Only returns those dataset modules on which user has at least one of *READ, WRITE, EXECUTE,* or *ADMIN*
```

Comment:
There will be no inconsistency in this case. Since we will enforce on all dataset modules first and make sure the user has ADMIN privileges on all of them